### DISCUSSION TITLE
Why is TVPassPort.com blocked?

### DISCUSSION THREAD

**Discussion by DaftMink** [StevenBlack/hosts#2446] [General]
As the title states "Why is TVPassPort.com blocked?". 
I've added this site to my whitelist, but this seems like a case of over blocking.

**Comment by DemurestPie** [comment]
@DaftMink I don't see this url anywhere in the list but I do see it in [developerdans list](https://www.github.developerdan.com/hosts/lists/ads-and-tracking-extended.txt)

**Reply by krystian3w** [reply]
Some exist in "private" file:

https://github.com/StevenBlack/hosts/blob/1cdb3159b92b61124eab5668d8ffa6ed841cc916/data/StevenBlack/hosts#L1443-L1445

**Comment by krystian3w** [comment]
https://github.com/StevenBlack/hosts/commit/a9fb35dcc29633679a9cd0b489a83b771675c209 - but title don't help well (`tvmds.tvpassport.com` + `www.tvpassport.com`). We do not know what specifically annoyed the author in 2018 to block them.


It is unlikely that they will be unlocked quickly and the author will move them to a secret file.


https://github.com/StevenBlack/hosts/commit/36fab9ad8f6518024e2c4def34a3b1f1a73e2f59#diff-7d576a3548225dd73f4a4510a2146eb562298ef5d1b26c28d1515f986b44725fR110 (`www.tvmds.tvpassport.com`) - mostly fallback for HOSTS and bugged Pi-Hole?
